modules/ROOT/pages/_partials/release_notes/sync-gateway-3-1-6-release-note.adoc

```diff
@@ -0,0 +1,34 @@
+== 3.1.6 -- May 2024
+
+Version 3.1.6 of Sync Gateway delivers the following features and enhancements:
+
+NOTE: For an overview of the latest features offered in Sync Gateway 3.1, see xref:whatsnew.adoc[New in 3.1].
+
+[#maint-3-1-6]
+=== Fixed Issues
+
+* https://issues.couchbase.com/browse/CBG-3683[CBG-3683 - (3.1.6 backport) Fixed PANIC error when closing channel cache after Mutation feed fails to start inside StartOnlineProcesses]
```

Comment:
I believe we should remove `(3.1.6 backport)` from the description?
(At least Dan often used to ask me to do that iirc)

Reply:
Sure, will do!